lims.rst

```diff
@@ -0,0 +1,137 @@
+LIMS API
+========
+
+
+Data structures
+---------------
+
+.. code:: python
+
+    @dataclass
+    class LIMSProposal:
+    """
+    """
+        pass
+        #TDB
+
+    @dataclass
+    class LIMSLaboratory:
+    """
+    """
+        pass
+        #TDB
+
+    @dataclass
+    class LIMSSession:
+    """
+    """
+        pass
+        #TDB
+
+
+
+API Functions
+-------------
+
+
+.. code:: python
+
+    from queue import DataCollectionTask
+
+    def get_proposal_by_username(str:username) -> List[LIMSProposal]:
+        """
+        :param str username: username
+        :returns: The list proposals belonging to username
+        :rtype: List[LIMSProposal]
+        """
+        pass
+
+    def get_proposal(str:code, str:number) -> LIMSProposal:
```

Comment:
Maybe get_proposal_by_code_and_number?

Reply:
well, I thought that proposals are usually or traditionally identified by code and number and that get_proposal_by_code_and_number was slightly long. I have, however nothing against it per say.